py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,40 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
+    format_device = '{name}'
+
+    class Meta:
+        def deprecate_function(config):
+            out = {}
+            if 'format_prefix' in config:
+                out['format'] = u'{}{{format_device}}'.format(config['format_prefix'])
+            return out
```

Comment:
currently it's hard but I'd like to make it easy.

As ever I have a branch that does this somewhere in my backlog of PRs

python3 has a nice way to do it but our threading model prevents it :( anything else is a nasty hack